esp-hal/MIGRATING-0.20.md

```diff
@@ -13,7 +24,7 @@ Instead of manually grabbing peripherals and setting up clocks, you should now c
      prelude::*,
 -    system::SystemControl,
  };
- 
+
```

Comment:
Technically this is now invalid diff sytanx 🙃

Reply:
How so?

Reply:
AFAIK the first character in a diff denotes what's going on, where a space means "unchanged line". It's just a markdown, so there's no difference here, but git might complain (if we cared about that here).